### PR TITLE
Show when stuff is an ESDA

### PR DESCRIPTION
**Pull request by MatMoore**
This only applies to a handful of assets, so in the case that a database is an ESDA, we will display this extra paragraph with a details component. This copy ideally needs to be rewritten in plain english at some point.

I've also removed metadata fields we have not populated yet (these will always display as blank) - this should be improved as part of https://github.com/ministryofjustice/find-moj-data/issues/423

We are deliberately not adding the ESDA notice on the search results page as we want this to be more concise. If we decide to show tags / glossary terms on the results page, we could potentially include ESDA there but at the moment we're not showing this.

Screenshot 2024-06-12 at 11 38 54

### CODE REVIEW

**Comment by alex-vonfeldmann** [comment]
so the correct spelling is MOJ not MoJ, so we'll need to fix that. Sorry my bad!

**Comment by MatMoore** [comment]
> so the correct spelling is MOJ not MoJ, so we'll need to fix that. Sorry my bad!

Thanks, fixed this now

**Review by mitchdawson1982 — APPROVED** [review]
LGTM
